src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const result = {};
```

Comment:
Avoid variable namings like `result`. It will decrease code readability. Consider using a more descriptive name, such as `stylesObject` or `cssProperties`.